packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -554,9 +640,28 @@ describe('TransactionController', () => {
         getGasFeeEstimates: () => Promise.resolve({}),
         getPermittedAccounts: () => [ACCOUNT_MOCK],
         getSelectedAddress: () => ACCOUNT_MOCK,
+        getNetworkClientRegistry: () => ({
+          sepolia: {
+            configuration: {
+              chainId: SEPOLIA.chainId,
+            },
+          },
+          goerli: {
+            configuration: {
+              chainId: GOERLI.chainId,
+            },
+          },
+          'customNetworkClientId-1': {
+            configuration: {
+              chainId: '0xa',
+            },
+          },
+        }),
```

Comment:
should include mainnet

Reply:
working on this

Reply:
https://github.com/MetaMask/core/pull/3643/commits/81fcd605873eb74e6f05d47cb9f85a46a4d2d216